### ISSUE TITLE
Some pods don't get started properly

### ISSUE THREAD

**Issue by RHRolun** [opendatahub-io-contrib/airflow-on-openshift#7]
Installed the operator through the helm chart according to https://github.com/opendatahub-io-contrib/airflow-on-openshift#install-airflow-via---set on an RHPDS demo environment of OpenShift 4.12
Some pods start running properly but most don't:
![image](https://user-images.githubusercontent.com/113337503/221591687-0d7e556c-84a8-4cbc-9706-6cafb684a90c.png)

All have the same error:
```
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "airflow-postgresql.airflow" (172.30.137.62), port 5432 failed: Connection refused
Is the server running on that host and accepting TCP/IP connections?
```

**Comment by codekow** [comment]
This is an issue with the stateful set `airflow-postgresql`

```
create Pod airflow-postgresql-0 in StatefulSet airflow-postgresql failed error: pods "airflow-postgresql-0" is forbidden: unable to validate against any security context constraint: [provider "anyuid": Forbidden: not usable by user or serviceaccount, provider "pipelines-scc": Forbidden: not usable by user or serviceaccount, provider restricted-v2: .spec.securityContext.fsGroup: Invalid value: []int64{1001}: 1001 is not an allowed group, spec.containers[0].securityContext.runAsUser: Invalid value: 1001: must be in the ranges: [1000820000, 1000829999], provider restricted: .spec.securityContext.fsGroup: Invalid value: []int64{1001}: 1001 is not an allowed group, provider "nonroot-v2": Forbidden: not usable by user or serviceaccount, provider "nonroot": Forbidden: not usable by user or serviceaccount, provider "hostmount-anyuid":
```

**Comment by shalberd** [comment]
I made it running with input from the airflow helm charts maintainer, taking into account securitycontextconstraints, group and user ids and so on. I have a working delta of the values.yaml, just have not found time to submit a PR here yet. 
The list of values in the documentation here is incomplete so far, will submit a PR soon. **I highly recommend not using the mucked up postgres image that comes with the helm charts.**

Working together with @thesuperzapper helped a lot https://github.com/airflow-helm/charts/issues/578#issuecomment-1419954471

**Comment by shalberd** [comment]
Ok, here we go.

Airflow-scheduler has the git-sync in it, I believe. I am using that mechanism, working fine.

Here are my deltas to the default values.yaml, my working values of values.yaml on the left.

I prefer using the yaml approach over command-line passing of vars, especially due to weird bugs with special characters and all that.

Do make sure you replace the gid or uid values or runAsUser with the correct namespace-level value of GID and / or UID.
In my case, from the airflow namespace range, GID and UID were equal, makes sense, as Openshift standard.
```

# get project e.g. airflow, newly-created
PROJECT=$(oc project -q)

# get openshift uid/gid range
CHART_UID=$(oc get project ${PROJECT} -o jsonpath="{['metadata.annotations.openshift\.io/sa\.scc\.uid-range']}" | sed "s@/.*@@")
CHART_GID=$(oc get project ${PROJECT} -o jsonpath="{['metadata.annotations.openshift\.io/sa\.scc\.supplemental-groups']}" | sed "s@/.*@@")

#use this value in the sections of values.yaml

echo $CHART_UID
```

@codekow @RHRolun I used postgres crunchy operator instead of the standard postgres container coming with the helm charts, don't get too hung up on that, I believe you get the idea. **I didn't see any way to make that mucked up postgres image work on openshift, so I'd say just use Crunchy Postgres via OLM**. I mean, even in the charts, it says not recommended for serious use or so (the included postgres) :-) so ... postgressql.enabled false

Context should be clear from the screenshots. I used airflow 2.5.3

`helm upgrade --install airflow apache-airflow/airflow --namespace ${PROJECT} -f values_sven.yaml`

![Bildschirmfoto 2023-06-20 um 23 05 38](https://github.com/opendatahub-io-contrib/airflow-on-openshift/assets/21118431/e80febfe-496e-43f2-96f7-ad26ec44fce5)

![Bildschirmfoto 2023-06-20 um 23 06 03](https://github.com/opendatahub-io-contrib/airflow-on-openshift/assets/21118431/2b400a7a-2f22-4310-90f3-87345716bf2c)

![Bildschirmfoto 2023-06-20 um 23 09 24](https://github.com/opendatahub-io-contrib/airflow-on-openshift/assets/21118431/5c1a07e7-5d6f-4a76-8f59-e6ea026a6df1)

![Bildschirmfoto 2023-06-20 um 23 09 47](https://github.com/opendatahub-io-contrib/airflow-on-openshift/assets/21118431/be33a047-2664-4b75-9517-a486a4fe8e83)

![Bildschirmfoto 2023-06-20 um 23 10 19](https://github.com/opendatahub-io-contrib/airflow-on-openshift/assets/21118431/bc740068-0edc-434f-a00a-21c4ae5b6dba)

![Bildschirmfoto 2023-06-20 um 23 10 55](https://github.com/opendatahub-io-contrib/airflow-on-openshift/assets/21118431/9585dd00-d8d2-4a4e-bb9f-e84f27a25842)

![Bildschirmfoto 2023-06-20 um 23 11 20](https://github.com/opendatahub-io-contrib/airflow-on-openshift/assets/21118431/beed2fa0-2a35-4b6e-83f9-e0dc259e6a77)

![Bildschirmfoto 2023-06-20 um 23 12 45](https://github.com/opendatahub-io-contrib/airflow-on-openshift/assets/21118431/2bbcc1f2-eacc-49ce-83e3-f9aafbb4117f)

**Comment by codekow** [comment]
Issue addressed in PR #9
